edkrepo/common/pathfix.py

```diff
@@ -181,7 +181,8 @@ def expanduser(path):
     if sys.platform != 'win32':
         return os.path.expanduser(path)
 
-    path = os.fspath(path)
+    if sys.version_info >= (3, 6):
+        path = os.fspath(path)

```

Comment:
expanduser() was added because python 3.8.* changed the behavior of os.path.expanduser()  
https://github.com/tianocore/edk2-edkrepo/commit/eae57fe1539e106b376e6628251a0d9daee46725

So, if python 3.5 is in use, then we could just return os.path.expanduser() , similar to the ```if sys.platform != 'win32':``` check right above the os.fspath() line.